samples/bluetooth/st_ble_sensor/src/main.c

```diff
@@ -174,10 +179,13 @@ void main(void)
 
 	err = button_init();
 	if (err) {
-		LOG_ERR("Button init error: (err %d)", err);
+		ENDLESS_LOOP();
```

Comment:
Better to log the error then `return`.  It's OK to return from `main()`.

Same for the next two uses of `ENDLESS_LOOP()`.  Then get rid of that macro.